src/pages/saksbehandling/beregning/VisBeregning.tsx

```diff
@@ -15,8 +17,31 @@ import styles from './visBeregning.module.less';
 
 interface Props {
     beregning: Beregning;
+    forventetinntekt: number;
 }
 
+const fradragMedForventetinntekt = (
+    fragdragsArray: Array<Fradrag>,
+    forventetinntekt: number,
+    intl: IntlShape
+): Array<Fradrag> => {
+    const totalArbeidsinntekt = fragdragsArray
+        .filter((f) => f.type === Fradragstype.Arbeidsinntekt)
+        .reduce((acc, arbeidsfradrag) => acc + arbeidsfradrag.beløp, 0);
+    const andreFradrag = fragdragsArray.filter((f) => f.type !== Fradragstype.Arbeidsinntekt);
+
+    if (totalArbeidsinntekt >= forventetinntekt) {
+        return fragdragsArray;
+    }
+
+    andreFradrag.push({
+        type: 'Forventet inntekt',
+        beløp: forventetinntekt,
+        beskrivelse: intl.formatMessage({ id: 'display.brukerForventetinntekt' }),
+    });
+    return andreFradrag;
+};
```

Comment:
Ble ikke akkurat denne logikken gjort en annen plass også?

Reply:
ja, for beregning i backend. Etter demoen i dag, var det litt vanskelig å skille mellom hvis arbeidsinntekt eller forventet inntekt var faktisk brukt i beregning.Hadde da en prat med Agnethe, og dette er egentlig bare for å direkte vise forventet inntekt er lagt til som et fradrag, uten at saksbehandler skal ha mulighet til å legge forventet inntekt som en type (ref andre endringer)

Reply:
Aaah, stemmer!

Kunne gjort logikken på samme måte som vi gjorde der ved å bruke f.eks. https://gcanti.github.io/fp-ts/modules/Array.ts.html#partition